MinecraftClient/Program.cs

```diff
@@ -229,12 +230,37 @@ private static void InitializeClient()
                 if (Settings.DebugMessages)
                     Translations.WriteLine("debug.session_id", session.ID);
 
-                //ProtocolHandler.RealmsListWorlds(Settings.Username, PlayerID, sessionID); //TODO REMOVE
+                List<string> availableWorlds = new List<string>();
+                if (Settings.DisplayRealmsWorld)
+                    availableWorlds = ProtocolHandler.RealmsListWorlds(Settings.Username, session.PlayerID, session.ID);
 
                 if (Settings.ServerIP == "")
                 {
                     Translations.Write("mcc.ip");
-                    Settings.SetServerIP(Console.ReadLine());
+                    string addressInput = Console.ReadLine();
+                    if (Settings.DisplayRealmsWorld && addressInput.StartsWith("realms:"))

```

Comment:
Warn user when trying to use Realms but Realms is disabled in config?
```suggestion
                    if (!Settings.MinecraftRealmsEnabled && addressInput.StartsWith("realms:"))
                        ConsoleIO.WriteLineFormatted("Warning: Looks like you are trying to join a Realms world but Realms support is disabled in config."); // TODO translate

                    if (Settings.MinecraftRealmsEnabled && addressInput.StartsWith("realms:"))
```
... Or just abort when trying to connect to a Realms world when support is disabled:
```C#
if (addressInput.StartsWith("realms:"))
{
    if (Settings.MinecraftRealmsEnabled)
    {
        // Existing code
    }
    else
    {
        HandleFailure("Trying to join a Realms world but Realms support is disabled in config", false, null); // TODO translate
        return;
    }
}
```
DisconnectReason is set to null because AutoRelog cannot handle this error.